ci-operator/config/openstack-k8s-operators/ceilometer-operator/openstack-k8s-operators-ceilometer-operator-main.yaml

```diff
@@ -0,0 +1,38 @@
+base_images:
+  openstack-operator-ci-pre-commit:
+    name: tools
+    namespace: openstack-k8s-operators
+    tag: openstack-operator-ci-pre-commit
+binary_build_commands: make build
+build_root:
```

Comment:
You will need to point to repo's base root image. It is easier for devs to update golang and operator-sdk versions:

https://github.com/openshift/release/blob/ba8a7c3ff6838ea69f548ada6e96412482160281/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L10-L11

Reply:
ack, will fix.

Reply:
I think that you missed this change. Without the correct build_root image, any job running with `from: src` may fail